netman/core/switch_sessions.py

```diff
@@ -15,13 +15,16 @@
 import logging
 import threading
 
+from netman.adapters.memory_session_storage import MemorySessionStorage
 from netman.core.objects.exceptions import UnknownSession, SessionAlreadyExists
 
 
 class SwitchSessionManager(object):
-    def __init__(self, session_inactivity_timeout=60):
+    def __init__(self, session_storage=None, session_inactivity_timeout=60):
```

Comment:
When adding a new parameter add it at the end so you keep the backward compatibility if people was using SwitchSessionManager(30)